source/def-stdlib.js

```diff
@@ -694,6 +694,9 @@ if (doc) doc.children.push({
 				<code class="code">StereoAudio(leftBuffer, rightBuffer, sampleRate)</code> provides the same functionality as above, but allows playback of stereo audio.
 			</td></tr>
 			</table>
+			<p> Both constructors allow disabling checks that ensure the arrays only contain floats and are in the proper ranges. 
+			This is disabled by default and can be set by calling the constructor with an optional trailing <code class="code">false</code>.</p>
+			
 			<h3>Functions</h3>
 			<tscript>
 			function play(){}
```

Comment:
The documentation should mention that sound is playes asynchronously, i.e., in the background, while the call returns immedately and the program continues to run. The alternative would be to wait for the sound to complete. Sure, that's rarely what's needed, but to a beginner the behavior may still be surprising.